src/main.js

```diff
@@ -6,6 +6,8 @@ import router from "./router";
 import store from "./store";
 import "./registerServiceWorker";
 import vuetify from "./plugins/vuetify";
+import InstantSearch from "vue-instantsearch";
+import VueObserveVisibility from "vue-observe-visibility";
```

Comment:
Dont mount stuff onto Vue prototype with Vue.use in main.js to reduce initial load speed